.buildkite/scripts/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common.sh
+
+MAKEFILE=${1}
+
+check_is_arm
+
+add_bin_path
+with_go "${GOLANG_VERSION}"
+with_mage
+google_cloud_auth
+
+make -C go -f "${MAKEFILE}" build"${is_arm}"
+docker images --format "table {{.Repository}}:{{.Tag}}\t{{.Size}}" --filter=reference="${STAGING_IMAGE}/golang-crossbuild"
+docker images --format "table {{.Repository}}:{{.Tag}}\t{{.Size}}" --filter=reference="${DOCKER_REGISTRY}/beats-dev/golang-crossbuild"
```

Comment:
Tried to do that, but there is no easy and fast way to do that - it will require to change almost all makefiles in the project. Taking into account that .ci will be removed quite soon, I think it's not wise to commit to that at the moment.